### PR TITLE
feat(bunyan support): add bunyan logging instrumentation

### PR DESCRIPTION
**Pull request by adavidai**

### CODE REVIEW

File: src/events/bunyan.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+/**
+ * Wrap bunyan logs
+ * @param {Function} wrappedFunction The function to wrap from bunyan
+ * @returns {function} emit wrapper function
+ */
+function emitWrapper(wrappedFunction) {
+    return function internalEmitWrapper(rec, ...args) {
+
+        if (tracer.getLogId() === null) {
```
Comment (galbash) [inline src/events/bunyan.js]: might be better to call `getLogId()` once and store it in a variable

Comment (galbash) [reply src/events/bunyan.js]: also might be better just checking `if(log_id)` and not comparing only to null

**Review by galbash — CHANGES_REQUESTED** [review]

**Comment by ronyis** [comment]
I think we should patch the logs only when the user configures to do so.
This flag should be also sent as part of the trace when set.

**Comment by ranrib** [comment]
> I think we should patch the logs only when the user configures to do so.
> This flag should be also sent as part of the trace when set.

I would prefer to have a flag to disable it, and enable it by default for everyone

**Review by galbash — APPROVED** [review]

**Pull request merged**